### DISCUSSION TITLE
Help with MacOS

### DISCUSSION THREAD

**Discussion by K7IU** [AlexandreRouma/SDRPlusPlus#897] [Q&A]
I am not familiar with GitHub but I've managed to get on.  Can someone please help me with some EASY to follow instructions on getting sdr++ loaded and working on my Mac mini?  When I downloaded a .pkg and tried to run it from terminal, I got the following:
dyld: Library not loaded: /usr/local/opt/glew/lib/libGLEW.2.2.dylib
  Referenced from: /usr/local/bin/sdrpp
  Reason: image not found
Abort trap: 6

Any help would be appreciated.

**Comment by AlexandreRouma** [comment]
If you're on an intel mac mini, install the nightly: https://www.sdrpp.org/nightly, everything needed is included.
If you're on M1, I guess try it with rosetta but it'll perform like trash.
Even developers struggle to build it on M1, I can't really make any kind of "simple" tutorial to do it.

**Reply by michaelgeorge001** [reply]
I've been using the nightly builds on an M1, and I have say I think it performs admirably. I've worked for companies that sold commercial software that performed a lot worse.

Considering you've only been developing this package for only a few years, and going to University at the same time, you're doing a great job.

-Mike-

**Comment by K7IU** [comment]
I’m an old guy and I see no nightly to install on the link you sent.  Could you be more detailed on what I  need to select?

Sent from my iPad
And YES it's cool!

**Reply by AlexandreRouma** [reply]
The link redirects you to the latest nightly on GitHub, scroll down to "Artifacts" and click one called `sdrpp_macos_intel`.

**Reply by fern244** [reply]
For some reason it's not allowing me to download the artifact..

**Comment by davekwright** [comment]
I tried build #833 (?) from the nightly on macOS 11.7 (20G817) and it crashes. Can send report if it will help.

_Dave KC6UPS

**Comment by davekwright** [comment]
Grant,
You can see the nightly build files without an account on git, but if you want to download any of them you must be logged in to your own github account...
